File: Exercise9.py
#Exercise 9: Paint Calculator

import math
length = int(input("Enter length: "))
width = int(input("Enter width: "))
area = length*width
gallons_needed = math.ceil(area/350)
print(f"You will need to purchase {gallons_needed} gallons of paint to cover {area} square feet.")

File: rb/Exercise9.rb
print "Enter length: "
length = gets.chomp.to_i
print "Enter width: "
width = gets.chomp.to_i
area = length * width 
gallons_needed = (area.to_f/350).ceil
puts "You will need to purchase #{gallons_needed} gallons of paint to cover #{area} square feet."